src/rpc.rs

```diff
@@ -267,6 +288,20 @@ impl Connection {
         Ok(unspent_from_status(&self.query.status(&script_hash[..])?))
     }
 
+    fn blockchain_scripthash_listcoloredunspent(&self, params: &[Value]) -> Result<Value> {
```

Comment:
Does this RPC support `color_id` argument that is described in protocol.md? 

I could not find the place where gets `params.get(1)`.

Reply:
Fixed at 166f8e45ae826b179ce8cf320d5676d294017af8